healthcheck/healthcheck.py

```diff
@@ -1,5 +1,5 @@
 import abc
-import os.path
+import utils
```

Comment:
Prefer absolute imports to relative ones. Especially implicit relative ones, like this. It should be `healthcheck.utils`.